lib/config.js

```diff
@@ -1,39 +1,64 @@
-// All imports in this file must have been installed from the trimet-mod-otp
-// project.
-import React from 'react'
-
 /**
- * To perform some custom operations, create a file like this one and reference
- * it in the config.yml entry `jsConfigFile`.
+ * To perform some custom operations, create a file like this one and specify it
+ * during build/start to override this default version of the file:
+ *    yarn start --env.JS_CONFIG=/path/to/config.js
  *
  * Caveats: This file cannot import the following:
  * - other files
  * - modules not installed in the node_modules of this project
- *
+ * - see config-example.js for more options on icon customization
+ */
+
+// All imports in this file must have been installed from the trimet-mod-otp
+// project.
+import React from 'react'
+
+// Import the base icon set use/customize.
+import { BiketownIcon, LegIcon, StandardModeIcon, StandardStreetcar } from '@opentripplanner/icons'
+
+// If you need to build a custom ModeIcon component,
+// below is an example using ClassicModeIcon (icons in "classic" OTP-react-redux) as base.
+// (You will need to include your own icon files as well.)
+//
+// import { ClassicModeIcon } from '@opentripplanner/icons'
+//
+// const MyModeIcon = ({ mode, ...props }) => {
+//   // Override icons for as few or as many OTP modes as needed.
+//   if (mode === 'RAIL') return <MyRailIcon {...props} />
+//   if (mode === 'TRANSIT') return <MyTransitIcon {...props} />
+//   // etc.
+//
+//   // For the remaining icons, just fall back to ClassicModeIcon.
+//   return <ClassicModeIcon mode={mode} {...props} />
+// }
+
+// You can also customize icons for specific itinerary legs.
+// Here, we build a LegIcon component with TriMet-specific logic.
+const TriMetLegIcon = ({ leg, ...props }) => {
+  if (leg.routeLongName && leg.routeLongName.startsWith('Portland Streetcar')) {
+    return <StandardStreetcar {...props} />
+  }
+  if (leg.rentedBike) {
+    if (leg.from.networks && leg.from.networks[0] === 'GBFS') {
+      return <BiketownIcon {...props} />;
+    }
+    return <StandardModeIcon mode='BICYCLE' {...props} />;
```

Comment:
Let's import the `StandardBike` component instead of the `StandardModeIcon` in order to avoid importing a whole bunch of other icons that aren't needed.

Reply:
Updated as of 5ea9f8cc8d313c869408c8101a200c21a68eea1f